src/docs/admin/references/properties/scheduler.properties

```diff
@@ -43,7 +43,7 @@ pa.scheduler.home=.
 # Example https://myserver.mydomain.com/catalog
 #pa.catalog.rest.public.url=
 
-# Set this property to define the public url of Service Automation (e.g. if behind a reverse proxy or accessed through a domain)
+# Set this property to define the public url of the cloud automation service (e.g. if behind a reverse proxy or accessed through a domain)
```

Comment:
Cloud Automation

Reply:
this is downloaded automatically from the scheduling scheduler.properties file. Changes need to be done in scheduling.

Reply:
I mean change in the comment cloud automation => Cloud Automation, not the property

Reply:
I understand, but I cannot change it here. scheduler.properties is automatically downloaded when the project is built.

Reply:
ok